readme.org

```diff
@@ -174,16 +174,32 @@ more.
 
         Any of those keys can be set to be a prefix key.
 
-   7. Prefix keys: ~g~ and ~z~ are the ubiquitous prefix keys.
+   8. Prefix keys: ~g~ and ~z~ are the ubiquitous prefix keys.
 
       - ~g~ generally stands for "go" and is best used for movements.
       - ~z~ is used for scrolling, folding, spell-checking and more.
 
-   8. Macro and action keys
+   9. Macro and action keys
 
       - ~@~, ~q~
       - ~.~
 
+   10. Ensure terminal compatibility without sacrificing GUI key bindings.
+
+       - Tab key
+         - Tab key is recognized as ~<tab>~ in GUI and ~TAB~ in terminal.
+           ~TAB~ equals ~C-i~.
+         - Since ~C-i~ is bound to jumping forward for vim compatibility,
+           bind the functions to which ~<tab>~ and ~<backtab>~ are bound on
+           vacant keys for terminal compatibility.
+       - Enter key
```

Comment:
I think this we can keep the Enter key documentation and leave the tab key changes out for now until we come to a better consensus.